src/components/accessibility-disability-simulator/README.md

```diff
@@ -0,0 +1,33 @@
+# d2l-labs-accessibility-disability-simulator
+
+A wrapper component that can be used to simulate how users with different forms of disabilities would experience the content it's wrapped around.
```

Comment:
```suggestion
A wrapper component that can be used to simulate how users with different forms of disabilities would experience the content within its slot.
```
It might be helpful to get slightly technical and specify about the slot (mostly since a couple definitions below get a bit ambiguous about slot vs line wrapping)